app/assets/src/components/views/compare/SamplesHeatmapView/SamplesHeatmapView.jsx

```diff
@@ -557,6 +557,7 @@ class SamplesHeatmapView extends React.Component {
       allData,
       addedTaxonIds,
       notifiedFilteredOutTaxonIds,
+      newestTaxonId,
```

Comment:
What happens if the user adds more than one taxa? Isn't that possible?

Reply:
The behavior I currently have implemented is that it will only scroll to the taxon once the Add Taxon dropdown is closed, so it will scroll to the most recently added one (the last one selected before the dropdown was closed).

Alternatively, we could have the heatmap scroll to the taxon as soon as it's selected, but I feel it could be a little jarring if the heatmap is constantly scrolling if the user is adding multiple taxa. This is something I'll check with @happyimadesignr !

Reply:
Yeah, makes sense. The first one given the order on the heatmap might make more sense. But definitely check with Jenn.